iocore/eventsystem/IOBuffer.cc

```diff
@@ -80,6 +80,7 @@ auto
 make_buffer_size_parser()
 {
   return [l = swoc::Lexicon<int>{
+    swoc::Lexicon<int>::with_multi{
```

Comment:
Does this extra `{` have a new closing for it? Unless I'm missing it, that would explain the format issues.